website/siteConfig.js

```diff
@@ -15,12 +15,12 @@ const users = require('./data/users.json');
 const siteConfig = {
   title: 'FDC3', // Title for your website. THIS CHANGES THE TITLE TAGS FOR THE WEBSITE (LOOK IN BROWSER TAB AT TOP)
   tagline: 'Open standards for the financial desktop', //THIS CHANGES THE TITLE TAGS FOR THE WEBSITE (LOOK IN BROWSER TAB AT TOP)
-  url: 'https://grizzwolf.github.io/FDC3/',
-  //cname: 'fdc3.finos.org',
-  baseUrl: '/FDC3/',
+  url: 'https://fdc3.finos.org',
```

Comment:
Ah. I see what happened now. This is a thorny problem to solve, it happened to me to. I think it is an unresolved issue on Docusaurus, how to target the same config & website at different URLs.

👍

Reply:
yes - that's better than my explanation. It was frustrating when we couldn't get it running, and figured out the base URL change (when forking in personal repos). Let me know if any issues other than that.